content/hardware/04.pro/boards/portenta-x8/tutorials/multi-protocol-gateway/content.md

```diff
@@ -35,15 +35,15 @@ In this tutorial we will go through the steps on how to setup both the Linux and
 - USB-C cable (either USB-C to USB-A or USB-C to USB-C)
 - Wi-Fi Access Point with Internet Access
 - 868-915 MHz antenna with SMA connector
-- ADB or SSH. [Check how to connect to your Portenta X8](docs.arduino.cc/tutorials/portenta-x8/out-of-the-box#controlling-portenta-x8-through-the-terminal)
+- ADB or SSH. [Check how to connect to your Portenta X8](https://docs.arduino.cc/tutorials/portenta-x8/out-of-the-box)
```

Comment:
Why the anchor has been removed? 🤔

Reply:
It was not working for me, it was giving 404. If you can make it work even better :)

Reply:
sure